internal/cmd/mongodbflex/user/update/update_test.go

```diff
@@ -249,20 +252,18 @@ func TestBuildRequest(t *testing.T) {
 		expectedRequest mongodbflex.ApiPartialUpdateUserRequest
 	}{
 		{
-			description: "update database only",
-			model:       fixtureInputModel(func(model *inputModel) {}),
-			expectedRequest: fixtureRequest().PartialUpdateUserPayload(mongodbflex.PartialUpdateUserPayload{
-				Database: utils.Ptr("default"),
-			}),
+			description:     "base",
+			model:           fixtureInputModel(),
+			expectedRequest: fixtureRequest(),
 		},
 		{
-			description: "update roles only",
+			description: "update database only",
```

Comment:
aren't we now missing the "update roles only" test case?  Why did you remove it?